src/main/kotlin/util/DbUtil.kt

```diff
@@ -102,22 +103,36 @@ private val SerialDescriptor.simpleName
 private val SerialDescriptor.elementsRange
     get() = 0..elementsCount.dec()
 
-inline fun <reified T : Any> cascadeDeleteCollectionTrigger(tableName: String): String = run {
+inline fun <reified T : Any> cascadeDeleteCollectionTrigger(
+    tableName: String,
+    operation: String,
+    returnValue: String,
+): String = run {
     val collectionPaths = T::class.serializerOrNull()?.descriptor?.collectionPaths() ?: emptyList()
-    "".takeIf { collectionPaths.isEmpty() } ?: """  
-        CREATE OR REPLACE FUNCTION trigger_for_$tableName() 
-        RETURNS TRIGGER LANGUAGE PLPGSQL AS ${'$'}trigger_for_$tableName${'$'}
+    createTrigger(tableName, collectionPaths, operation, returnValue)
+}
+
+fun createTrigger(
```

Comment:
mb make 'createTrigger' fun as extention fun.
I think it's more readable  - creating trigger for table 
```suggestion
    tableName.createTrigger(collectionPaths, operation, returnValue)
}

fun String.createTrigger(
```